### PR TITLE
replace import with media query for dark and light variables

### PR DESCRIPTION
**Pull request by Jieiku**
I recently finished a Zola Themes Benchmarks page https://github.com/Jieiku/zola-themes-benchmarks/blob/main/README.md

I noticed YellowLabTools flagged the use of import:

In abridge I handle the dark/light variables using media queries, so I tried it out in your theme and it appears to work properly.

### CODE REVIEW

**Comment by ayushnix** [comment]
Thanks!

**Comment by Jieiku** [comment]
No problem, also I worked out how to override colors in Zola for those that use a theme as a submodule and do not want to have to directly edit the theme... you can see that on my refactor branch of abridge:

https://github.com/Jieiku/abridge/tree/refactor

I am just waiting for the next version of zola to release before I merge my refactor branch, other than that it's done.

(I created this pull request before working that bit out)

**Comment by ayushnix** [comment]
To be honest, I've lost interest in Zola after facing [issues with writing Markdown](https://github.com/nvim-treesitter/nvim-treesitter/issues/2916) in my editor and realizing that [even CommonMark is hard to parse](https://www.johnmacfarlane.net/beyond-markdown.html). I found [djot](https://djot.net/) and I'm now interested in building my own static site with djot documents using [soupault](https://soupault.app/) SSG because I doubt zola will support additional markup languages.

This is one of the reasons I couldn't get to merging this PR. Sorry about the delay. 😅 I'll probably archive this repo in the near future as soon as I start work on my new SSG.

**Comment by Jieiku** [comment]
Hey that's cool too and your not the first person I have heard mention soupault, hope it works out for ya, even if it don't, trying new things keeps you learning!
